osu.Game.Tests/Visual/Gameplay/TestSceneBeatmapOffsetControl.cs

```diff
@@ -45,6 +45,44 @@ public void TestTooShortToDisplay()
             AddAssert("No calibration button", () => !offsetControl.ChildrenOfType<SettingsButton>().Any());
         }
 
+        /// <summary>
+        /// When a beatmap offset was already set, the calibration should take it into account.
```

Comment:
This xmldoc is probably even worse than the other one because it's so vague ("take it into account"). I'd just copy & paste across honestly.

```suggestion
        /// If we already have an old score with enough hit events and the new score doesn't have enough, continue displaying the old one rather than showing the user "play too short" message.
```

Reply:
Was incorrectly copied from https://github.com/ppy/osu/pull/33626/files#diff-000cf7efd3369c3c5cba906d9c1a3ac1f620e810b2cf6207b4f6796f08c1b255L139